modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,6 +15,136 @@
+* Enhancement - Make server side space templates production ready. Fixes several smaller bugs and adds some improvements to space templates, introduced with #8558 #8723
+* Enhancement - Allow to resolve public shares without the ocs tokeninfo endpoint. Instead of querying the /v1.php/apps/files_sharing/api/v1/tokeninfo/ endpoint, a client can now resolve public and internal links by sending a PROPFIND request to /dav/public-files/{sharetoken} authenticated clients accessing an internal link are redirected to the "real" resource (`/dav/spaces/{target-resource-id} authenticated clients are able to resolve public links like before. For password protected links they need to supply the password even if they have access to the underlying resource by other means. unauthenticated clients accessing an internal link get a 401 returned with WWW-Authenticate set to Bearer (so that the client knows that it need to get a token via the IDP login page. unauthenticated clients accessing a password protected link get a 401 returned with an error message to indicate the requirement for needing the link's password. #8858 #8926 cs3org/reva#4653
+* Enhancement - Configurable claims for auto-provisioning user accounts. We introduce the new environment variables "PROXY_AUTOPROVISION_CLAIM_USERNAME", "PROXY_AUTOPROVISION_CLAIM_EMAIL", and "PROXY_AUTOPROVISION_CLAIM_DISPLAYNAME" which can be used to configure the OIDC claims that should be used for auto-provisioning user accounts. The automatic fallback to use the 'email' claim value as the username when the 'preferred_username' claim is not set, has been removed. Also it is now possible to autoprovision users without an email address. #8635 #6909 #8952
+* Enhancement - Theme Processing and Logo Customization. We have made significant improvements to the theme processing in Infinite Scale. The changes include: Enhanced the way themes are composed. Now, the final theme is a combination of the built-in theme and the custom theme provided by the administrator via WEB_ASSET_THEMES_PATH and WEB_UI_THEME_PATH. - Introduced a new mechanism to load custom assets. This is particularly useful when a single asset, such as a logo, needs to be overwritten. - Fixed the logo customization option. Previously, small theme changes would copy the entire theme. Now, only the changed keys are considered, making the process more efficient. - Default themes are now part of ocis. This change simplifies the theme management process for web. These changes enhance the robustness of the theme handling in Infinite Scale and provide a better user experience. #8966 #9133
+* Enhancement - Add command to check ocis backup consistency. Adds a command that checks the consistency of an ocis backup. #9238
+* Enhancement - Web server compression. We've added a compression middleware to the web server to reduce the request size when delivering static files. This speeds up loading times in web clients. owncloud/web#7964 #9287
```

Comment:
@JammingBen has also made more really noticeable improvements regarding the web ui initial loading time / performance. He had some numbers for the lighthouse score improvement for the web ui, I think it was something like before: 50, after: 94 (out of 100).